packages/web3-eth/types/index.d.ts

```diff
@@ -0,0 +1,268 @@
+    raw: string;
+    tx: Transaction;
+}
+
+export interface PastLogsOptions {
+    fromBlock?: number | string;
+    toBlock?: number | string;
+    address: string | string[];
+    topics?: Array<string | string[]>;
+}
+
+export interface Logs {
+    fromBlock?: number
+    address?: string
+    topics?: Array<string | string[]>
+}
+
+export interface Log {
+    address: string;
+    data: string;
+    topics: Array<string | string[]>;
+    logIndex: number;
+    transactionIndex: number;
+    transactionHash: string;
+    blockHash: string;
+    blockNumber: number;
+}
+
+export interface Subscribe<T> {
+    subscription: {
+        id: string
+        subscribe(callback?: (error: Error, result: Subscribe<T>) => void): Subscribe<T>
+        unsubscribe(callback?: (error: Error, result: boolean) => void): void | boolean
+        arguments: object
+    }
+    on(type: 'data', handler: (data: T) => void): void
+    on(type: 'changed', handler: (data: T) => void): void
+    on(type: 'error', handler: (data: Error) => void): void
+}
+
+/** DELETE ONCE personal is complete */
+
+export class Personal {
+    newAccount(password: string, callback?: (error: Error, result: boolean) => void): Promise<boolean>
+    getAccounts(callback?: (error: Error, result: string[]) => void): Promise<string[]>
+}
+/** END */
+
+/**            MOVE ALL BELOW TO WEB3-CORE ONCE TYPES COMPLETE FOR CLEAR UP    !!! */
+
+export interface PromiEvent<T> extends Promise<T> {
+    once(type: 'transactionHash', handler: (receipt: string) => void): PromiEvent<T>
+    once(type: 'receipt', handler: (receipt: TransactionReceipt) => void): PromiEvent<T>
+    once(type: 'confirmation', handler: (confNumber: number, receipt: TransactionReceipt) => void): PromiEvent<T>
+    once(type: 'error', handler: (error: Error) => void): PromiEvent<T>
+    once(type: 'error' | 'confirmation' | 'receipt' | 'transactionHash', handler: (error: Error | TransactionReceipt | string) => void): PromiEvent<T>
+    on(type: 'transactionHash', handler: (receipt: string) => void): PromiEvent<T>
+    on(type: 'receipt', handler: (receipt: TransactionReceipt) => void): PromiEvent<T>
+    on(type: 'confirmation', handler: (confNumber: number, receipt: TransactionReceipt) => void): PromiEvent<T>
+    on(type: 'error', handler: (error: Error) => void): PromiEvent<T>
+    on(type: 'error' | 'confirmation' | 'receipt' | 'transactionHash', handler: (error: Error | TransactionReceipt | string) => void): PromiEvent<T>
+}
+
+export interface Transaction {
+    from?: string | number;
+    to?: string;
+    gasPrice?: string;
+    gas?: number | string;
+    value?: number | string;
+    chainId?: number;
+    data?: string;
+    nonce?: number;
+    v?: string;
+    r?: string;
+    s?: string;
+}
+
+export interface SignedTransaction {
+    messageHash?: string;
+    r: string;
+    s: string;
+    v: string;
+    rawTransaction?: string;
+}
+
+export interface Sign extends SignedTransaction {
```

Comment:
It does not really extend from SignedTransaction it's just an encrypted string

Reply:
was not even used so remove completely.